logging/logging-processor/src/main/java/org/infinispan/logging/processor/InfinispanLoggingProcessor.java

```diff
@@ -0,0 +1,116 @@
+package org.infinispan.logging.processor;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import javax.annotation.processing.AbstractProcessor;
+import javax.annotation.processing.Processor;
+import javax.annotation.processing.RoundEnvironment;
+import javax.annotation.processing.SupportedAnnotationTypes;
+import javax.annotation.processing.SupportedSourceVersion;
+import javax.lang.model.SourceVersion;
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.element.AnnotationValue;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.TypeElement;
+import javax.tools.Diagnostic;
+import javax.tools.StandardLocation;
+import javax.xml.stream.XMLStreamException;
+
+import org.infinispan.logging.processor.report.XmlReportWriter;
+import org.jboss.logging.processor.apt.ProcessingException;
+import org.kohsuke.MetaInfServices;
+/**
+ * InfinispanLoggingProcessor.
+ *
+ * @author Durgesh Anaokar
+ * @since 13.0
+ */
+@SupportedSourceVersion(SourceVersion.RELEASE_8)
+@SupportedAnnotationTypes({InfinispanLoggingProcessor.DESCRIPTION})
+@MetaInfServices(Processor.class)
+public class InfinispanLoggingProcessor extends AbstractProcessor {
+   static final String DESCRIPTION = "org.infinispan.logging.annotations.Description";
+
+   @Override
+   public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
+      for (TypeElement annotation : annotations) {
+            createReportXml(roundEnv, annotation);
+      }
+      return true;
+   }
+
+   private void createReportXml(RoundEnvironment roundEnv, TypeElement annotation){
```

Comment:
The indentation is wrong in this method. Please use the style templates in your ide from the `ide-settings`. It will make this a LOT easier :slightly_smiling_face: